frontend/components/auction/collateral/CollateralAuctionExecutionBlock.vue

```diff
@@ -10,13 +10,21 @@
             show-different-wallet-info
         />
         <div class="flex flex-col md:flex-row md:space-x-4 justify-end flex-wrap mt-4">
-            <CollateralAuctionExecuteWithOtherWalletBlock
-                :disabled="disabled || isLoading || state === 'executed'"
+            <ExecuteWithOtherWalletModal
+                :is-shown.sync="isExecuteToAnotherWalletModalShown"
                 :default-wallet="walletAddress"
-                :is-loading="state === 'loading'"
                 class="pb-3"
                 @execute="executeWithOtherWallet"
+                @close="closeExecuteToOtherWalletModal"
```

Comment:
Why do you need close event if you have synced prop?

Reply:
It is needed for the cancel button in the footer.

Even in their docs they create a custom method when declaring a custom footer:
https://1x.antdv.com/components/modal/

Reply:
> if you have synced prop

The issue is not related to the use of modal component, but to the use of a synced prop. Synced prop let you modify its value from the child. Instead of emitting `close` event, the component should emit `'update:isShown', false`. This will remove the need of an extra event and an extra event handler